services/enrichment/fields/locations/src/lib/computeId.js

```diff
@@ -0,0 +1,9 @@
+import crypto from 'crypto';
+
+export const computeId = ({ computedKey, projectId }) =>
```

Comment:
Can we use this helper in `services/value-store/projects/src/lib/SaveStream.js` in order to ensure the 2 always match?

Reply:
We could/should have a `lib` folder at the root of the project where we store all our helpers 👍